runtime/runtime.go

```diff
@@ -99,6 +99,10 @@ type Params struct {
 	// CertPool holds the CA certs trusted by the OPA server.
 	CertPool *x509.CertPool
 
+	// MinVersion contains the minimum TLS version that is acceptable.
+	// If zero, TLS 1.2 is currently taken as the minimum.
+	MinTLSVersion string
```

Comment:
Would it make sense to use the [`tls` constants](https://golang.org/pkg/crypto/tls/#pkg-constants) here instead? The cmd/run.go could make the translation; abort with an error if the passed version was invalid, and `runtime/runtime.go` could use whatever was passed as-is?

Reply:
Done